cicd/pipelines/reducing-pipelines-resource-consumption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="reducing-pipelines-resource-consumption"]
+= Reducing resource consumption of pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: reducing-pipelines-resource-consumption
+
+toc::[]
+
+
+If you use clusters in multi-tenant environments you must control the resource (cpu and memory) consumption per namespace or resource in order to prevent one application from consuming more than it should and creating problems for other applications.
+
+{pipelines-title} use resource quota limits and limit ranges of the project in which they are executed to define the final resource limits that are set on the resulting pods.
+
+To restrict resource consumption in your project, you can:
+
+* xref:../../applications/quotas/quotas-setting-per-project.html[Set and manage resource quotas] to limit the aggregate resource consumption in your project.
```

Comment:
Ending the sentence with "... in your project." sounds redundant because of the stem sentence before the bullet points.